languages/elixir/exercises/concept/numbers/.meta/example.ex

```diff
@@ -0,0 +1,21 @@
+defmodule AssemblyLine do
+
+  @spec success_rate(integer()) :: float()
+  defp success_rate(speed) do
+    cond do
+      speed >= 9 -> 0.77
+      speed >= 5 -> 0.9
+      true       -> 1.0
+    end
+  end
+
+  @spec production_rate_per_hour(integer()) :: float()
+  def production_rate_per_hour(speed) do
+    221 * speed * success_rate(speed)
+  end
+
+  @spec working_items_per_minute(integer()) :: integer()
+  def working_items_per_minute(speed) do
+    (production_rate_per_hour(speed) / 60) |> trunc()
+  end
```

Comment:
It seems artificial to have this rate be an integer, especially since it deals with the probability that a car will be produced successfully. Could we invert the public functions return types and reframe `production_rate_per_hour` as `minimum_total_cars_produced_per_hour` (an integer) and `working_items_per_minute` as something like `average_working_cars_produced_per_hour` (a float)?

Reply:
I was following the existing narrative put forward by the `csharp` and `fsharp` tracks.  I don't have strong feelings about this and would be happy to follow convention if we decide otherwise.

C# and F# have this feature in their as they are strongly typed that require explicit conversion from one to the other, and I thought that one thing to highlight is that elixir doesn't.

Reply:
In a vacuum, I like my suggestion a lot more, but there is value in having some uniformity between the tracks where it makes sense. Maybe others will have some opinions?

Reply:
In a vacuum, agreed.  Let's wait for others?

@ErikSchierboom how much have other tracks been holding to the same designs?

Reply:
I don't think many tracks have made copies of this. And I like the changes. I'd say: try to apply them and see if it makes sense, and ping me when the code has been updated.

Reply:
I'll work on this tomorrow.

Reply:
see commit